lib/elixir/lib/task.ex

```diff
@@ -813,17 +832,18 @@ defmodule Task do
 
   """
   @spec await(t, timeout) :: term
-  def await(%Task{ref: ref, owner: owner} = task, timeout \\ 5000) when is_timeout(timeout) do
+  def await(%Task{ref: ref, owner: owner} = task, timeout \\ 5000)
```

Comment:
We will have to update the example given in this function. :)

Reply:
Done, I think the implementation I put in the example is ok? Any suggestions?

Reply:
Done, I think the implementation I put in the example is ok? Any suggestions?

Reply:
Done, I think the implementation I put in the example is ok? Any suggestions?